package.json

```diff
@@ -96,6 +96,8 @@
     "**/minimist": "^1.2.5",
     "**/node-jose/node-forge": "^0.10.0",
     "**/prismjs": "1.22.0",
+    "**/react-syntax-highlighter": "^15.3.1",
+    "**/react-syntax-highlighter/**/highlight.js": "^10.4.1",
```

Comment:
Forcefully resolves:

* `react-syntax-highlighter > highlight.js@^10.1.1` to `^10.4.1`
* `react-syntax-highlighter > lowlight > highlight.js@~10.1.0` to `^10.4.1`

See [`highlight.js` changelog](https://github.com/highlightjs/highlight.js/blob/master/CHANGES.md).